pkgs/by-name/uu/uutils-coreutils/package.nix

```diff
@@ -39,11 +54,22 @@ stdenv.mkDerivation (finalAttrs: {
       "CARGO=${lib.getExe cargo}"
       "PREFIX=${placeholder "out"}"
       "PROFILE=release"
+      "SELINUX_ENABLED=${if selinuxSupport && stdenv.hostPlatform.isLinux then "1" else "0"}"
       "INSTALLDIR_MAN=${placeholder "out"}/share/man/man1"
     ]
     ++ lib.optionals (prefix != null) [ "PROG_PREFIX=${prefix}" ]
     ++ lib.optionals buildMulticallBinary [ "MULTICALL=y" ];
 
+  env =
+    { }
+    // lib.optionalAttrs (selinuxSupport && stdenv.hostPlatform.isLinux) {
```

Comment:
```suggestion
    lib.optionalAttrs (selinuxSupport && stdenv.hostPlatform.isLinux) {
```
nit.

Reply:
This minimises future diffs. It's used in many expressions in nixpkgs.
I don't mind removing it if you feel really strongly about it though.

Reply:
I personally don't feel strongly about that, but I don't think we should circumvent the formatter by adding unneeded empty sets/lists.
Also, this is the very first time that I see this being done in nixpkgs.

I took the liberty of removing them to stick to the more consensual style.
I hope it's fine for you.

IIRC the formatting team is aware of this problem and is testing some solutions.